content/en/building/contact-management/contact-and-users-1.md

```diff
@@ -102,8 +105,7 @@ Once you submit, a new CHW Area will be created. On the right-hand you should se
 
 {{< figure src="new-chw-supervisor/new-person.png" link="new-chw-supervisor/new-person.png" class="right col-6 col-lg-8" >}}
 
-To create a primary contact for an existing Place (in this case, for the Health Facility that we created without a primary contact); select the Place and then select the **New Person** action.
-
+To create a primary contact for an existing Place (in this case, for the Health Facility that we created without a primary contact); select the Place and then select the Floating Action Button (FAB) that opens a menu with all actions, in the menu select the **New Person** action.
 
 {{< figure src="new-chw-supervisor/belongs-to.png" link="new-chw-supervisor/belongs-to.png" class="right col-6 col-lg-8" >}}
```

Comment:
The new screenshot (left) is clicking the "Full name" vs the old screenshot (right) is clicking "Belongs to".   I believe the new screenshot is wrong and the old is correct:
![image](https://github.com/user-attachments/assets/5d7e72ea-d641-4666-afde-b2478986ee25)

Reply:
Good catch! I moved the taping hand to the `Belongs To` field.